src/assets/Colors.js

```diff
@@ -1,25 +1,28 @@
 const COLORS = {
-    white: "#fff",
-    black: "#000",
-    lightblack: "#212121",
+  white: "#fff",
 
-    darkgrey: "#2F2F2F",
-    grey: "#9A9A9A",
-    lightgrey: "#E0E0E0",
+  black: "#000",
+  lightblack: "#212121",
 
-    greygreen1: "#ADB1A3",
-    greygreen2: "#B5BCA3",
-    greygreen3: "#C1CCA5",
-    greygreen4: "#CDE09B",
+  grey: "#9A9A9A",
+  darkgrey: "#2F2F2F",
+  lightgrey: "#E0E0E0",
 
-    green: "#648C1C",
-    lightgreen: "#CAF28D",
-    mutedgreen: "#9CA670",
-    darkgreen: "#557718",
-    forestgreen: "#2F6443",
+  greygreen1: "#ADB1A3",
+  greygreen2: "#B5BCA3",
+  greygreen3: "#C1CCA5",
+  greygreen4: "#CDE09B",
 
-    lightblue: "#82A2F4",
-    sandy: "#FFFCDC",
+  green: "#648C1C",
+  lightgreen: "#CAF28D",
+  mutedgreen: "#9CA670",
+  darkgreen: "#557718",
+  forestgreen: "#2F6443",
+
+  lightblue: "#82A2F4",
+
+  sandy: "#FFFCDC",
+  signUpBkg: "rgb(255, 253, 223)",
```

Comment:
let's keep all of these using hex, also give it a modular name so we can use it for other stuff